plugins/modules/mysql_db.py

```diff
@@ -543,37 +543,30 @@ def db_create(cursor, db, encoding, collation):
 
 
 def main():
+    argument_spec = mysql_common_argument_spec()
+    argument_spec.update(
+        name=dict(type='list', required=True, aliases=['db']),
```

Comment:
@Jorge-Rodriguez Why did you strip `login_user` and related vars there and what impact will it have ?

Reply:
@bmalynovytch They are not stripped, they come from `mysql_common_argument_spec()`. The collection was inconsistent, some modules were using the common spec in utils and some weren't.